src/main.rs

```diff
@@ -87,9 +95,13 @@ fn main() {
         });
 
         if let Err(e) = res {
-            println!("An error occured: {}", e);
+            println!("An error occured: {}\nresting {} Seconds", e, backoff_seconds);
```

Comment:
Whoops, @rnestler is right :) Sorry about that. `Sleeping for n seconds` would probably be the best solution.